.eslintrc.json

```diff
@@ -40,10 +40,12 @@
         "sourceType": "module",
         "ecmaFeatures": { "jsx": true }
       },
+      "plugins": ["@tanstack/query"],
       "extends": [
         "plugin:@typescript-eslint/recommended",
         "plugin:@typescript-eslint/recommended-requiring-type-checking",
-        "plugin:import/typescript"
+        "plugin:import/typescript",
+        "plugin:@tanstack/eslint-plugin-query/recommended"
```

Comment:
Nyt pitäisi ESLint toimia `tanstack`in kanssa myös. Säännöiksi asetin `tanstack`in säännökset mitä he suosittelevat